sphinx/parsers.py

```diff
@@ -52,7 +52,8 @@ class Parser(docutils.parsers.Parser):
     """
 
     def set_application(self, app: "Sphinx") -> None:
-        """set_application will be called from Sphinx to set app and other instance variables
+        """
+        set_application will be called from Sphinx to set app and other instance variables
```

Comment:
Why folding? In my understanding, the first line of the docstring can follow just after quotation marks.

Reply:
Ah I think I may have been trying to reduce line length (from 90 to 87 characters)? Happy for you (or me) to change this back if you prefer

Reply:
Thank you for your explanation. Now our line length limitation is 95. So reverting it is better. thank you for accepting.
https://github.com/sphinx-doc/sphinx/blob/f9941b9402821de2d60c1e94f9ac5cf0379640d0/setup.cfg#L29-L30